WS2012R2/lisa/remote-scripts/ica/NET_VerifyHotAddMultiNIC.sh

```diff
@@ -23,6 +23,99 @@
 
 
 expectedCount=0
+declare os_VENDOR os_RELEASE os_UPDATE os_PACKAGE os_CODENAME
+
+########################################################################
+# Determine what OS is running
+########################################################################
+# GetOSVersion
```

Comment:
Please source this function from utils.sh - include this in the xml definition and source it accordingly in the script.